contributing/code/standards.rst

```diff
@@ -167,6 +167,29 @@ Service Naming Conventions
 
 * A group name uses the underscore notation.
 
+.. _contributing-code-standards-comparisons:
+
+Comparisons
+-----------
+
+Use `identical comparison`_ when the expected value must match a specific type:
```

Comment:
This is the [original note](https://github.com/symfony/symfony-docs/pull/5403#discussion_r32434353) from @stof about identical comparisson.